nixos/modules/services/networking/twingate.nix

```diff
@@ -0,0 +1,28 @@
+{ config, lib, pkgs, ... }:
+
+with lib;
+
+let
+  cfg = config.services.twingate;
+
+in {
+
+  options.services.twingate = {
+    enable = mkEnableOption ''Twingate Client daemon'';
```

Comment:
```suggestion
    enable = mkEnableOption (lib.mdDoc "Twingate Client daemon");
```

Reply:
@SuperSandro2000, thanks for suggestion!